youTube_api_js2.6/test6.js

```diff
@@ -0,0 +1,253 @@
+var truth = true;
+var cardcount = 0;
+var cardId = 1;
+let videoData;
+var pointer=0;
+
+
+
+
+let list = document.getElementById("list");
+let btn = document
```

Comment:
use better names for variables, for ex, instead of btn could have used searchButton